client/src/components/BookEvent/index.js

```diff
@@ -15,125 +45,154 @@ class BookEvent extends React.Component {
-              name="time"
-              placeholder="Time:"
-            />
-            <input className="equipmentcost"
-              type="Number"
-              name="equipmentprice"
-              placeholder="Cost:"
-
-            />
-            <div className="total">
-            <h2 className="totalcost">Total Cost:</h2>
-            <input className="submitBtn"
-            type="submit"
-            >
-            </input>
-            </div>
-            </div>
-
+          <div className="BookEvent">
+            <form onSubmit={this.handleSubmitForm}>
+              <div className="container">
+                <div className="column1">
+                  <h2 className="h2">General</h2>
+                  <input
+                    className="input"
+                    type="text"
+                    name="title"
+                    placeholder="Event Name:"
+                    value={this.state.title}
+                    onChange={this.handleChange}
+                    required
+                  />
+                  <label className="label">Start Date:</label>
+                  <DatePicker
+                    className="bookdates"
+                    selected={this.state.start_date}
+                    onChange={this.handleChangeStartDate}
+                    name="start_date"
+                  />
+
+                  <input
+                    className="note"
+                    type="text"
+                    name="note"
+                    placeholder="Note:"
+                    value={this.state.note}
+                    onChange={this.handleChange}
+                  />
+                  <input
+                    className="input"
+                    type="text"
+                    name="capacity"
+                    placeholder="#Persons:"
+                    value={this.state.capacity}
+                    onChange={this.handleChange}
+                    required
+                  />
+                  <h2 className="h2">Lunch</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="lunch_note"
+                    placeholder="Description:"
+                    value={this.state.lunchnote}
+                    onChange={this.handleChange}
+                  />
+
+                  <h2 className="coffee">Coffee</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="coffee_note"
+                    placeholder="Description:"
+                    value={this.state.coffeenote}
+                    onChange={this.handleChange}
+                  />
+
+                  <h2 className="equipment">Equipment</h2>
+                  <input
+                    className="note"
+                    type="text"
+                    name="equipment_note"
+                    placeholder="Description:"
+                    value={this.state.equipmentnote}
```

Comment:
it would be better if you make the ```equipmentnote ``` variable much readable like ```equipmentNote```

Reply:
done